server/views/components/sample/sample.njk

```diff
@@ -1,6 +1,7 @@
 <div class="grid__cell">
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
-    <p>And some sample text</p>
+    <p>{{ text }}</p>
+    {% include "views/components/grid-placeholder.njk" %}
   </div>
 </div>
```

Comment:
Having each component in its own directory lets us add notes and context data to be displayed in the styleguide.